lib/rubygems/installer.rb

```diff
@@ -586,7 +586,7 @@ def shebang(bin_file_name)
     path = File.join gem_dir, spec.bindir, bin_file_name
     first_line = File.open(path, "rb") {|file| file.gets}
 
-    if /\A#!/ =~ first_line
+    if first_line&.start_with?("#!")
```

Comment:
This will bring first occurence of this syntax to RubyGems. Is that intentional? Wouldn't be `first_line.to_s.start_with` safer at least for now?

Reply:
I can make that change, to be less controversial in this PR.